docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
+---
+
+Under the main trinity of [Voluneteer teams](volunteers/teams.md),namely Outreach, Education & Personal Development,
+generally, we need to act and fulfill different roles in order to run the course smoothly and effectively.
+These roles also help to keep each member being engadged and organised.
+
+A volunteer can wear different hats on the way depending on the schedule and agenda, or maybe special circumstances.
+Everything is optional, there is no force if someone does not feel being comfortable to be responsible or to do something.
```

Comment:
> Everything is optional, there is no force if someone does not feel being comfortable to be responsible or to do something.

Suggest re-write as

"All roles and tasks are optional, and no one is forced, or coerced into to doing things that they do not feel comfortable with."

Reply:
Thank @christopherallanperry! It sounds way better :).

Reply:
Cheers Zoltan. I'll go down the rest of the doc in the morning - getting a bit late for it now.